cdh-web.yml

```diff
@@ -12,6 +12,7 @@
     - build_project_repo
     - build_virtualenv
     - install_local_settings
+    - build_npm
```

Comment:
Should this step be added to the cdh-web staging playbook also? (IDK how much we're still using that, but seems best to keep them in sync)

Reply:
couldn't hurt!